plugins/reviewers/index.js

```diff
@@ -216,7 +216,7 @@ export default class ReviewerPlugin extends BasePlugin {
       }
       const matches = REVIEWER_REGEX.exec(testSubject);
       if (!matches) return testSubject;
-      return matches[1];
+      return `${matches[1]}${process.env.GITHUB_USER_SUFFIX || ''}`;
```

Comment:
what's GITHUB_USER_SUFFIX and how does it get passed in?

Reply:
We need to define define that environment variable in the serverless config for the GHEC instance (https://github.com/gdcorp-uxp/pullie)